### PR TITLE
Change mention of conda (was: change conda to miniforge)

### PR DESCRIPTION
**Pull request by eteq**
This is really just a companion to astropy/astropy#16555 to synchronize what's on the web site with whatever that settles on.  Please have discussion about what we should actually recommend in astropy/astropy#16555 so that we don't spread it around multiple threads. 

Rendered doc: https://output.circle-artifacts.com/output/job/fc974b2f-31be-45e0-a824-116fb2da9647/artifacts/0/html/index.html

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -119,8 +119,8 @@
 		<h1 id="install-astropy">Install Astropy<a class="paralink" href="#install-astropy" title="Permalink to this headline">¶</a></h1>
         There are a number of ways of installing the latest version of the astropy core package. If you normally use <code>pip</code> to install Python packages, you can do:
         <p/><pre>pip install astropy[recommended] --upgrade</pre>
-        If instead you normally use <code>conda</code>, you can do:
-        <p/><pre>conda install -c conda-forge astropy</pre>
+        If instead you normally use <code>conda</code> or <code>miniforge</code>, you can do:
+        <p/><pre>miniforge install astropy</pre>
```
Comment (pllim) [inline index.html:123]: I think it is still `conda install`?
```suggestion
        <p/><pre>conda install -c conda-forge astropy</pre>
```

**Review by pllim — COMMENTED** [review]
astropy/astropy#16555 was superseded by https://github.com/astropy/astropy/pull/16987 and I don't see any usage of `miniforge install` over there.

**Comment by bsipocz** [comment]
>  I don't see any usage of miniforge install over there.

That is a very very unfortunate outcome. Switching from conda to miniforge was a clear recommendation coming out of the NF summit. Those PRs ended up in the usual bikeshedding of pip vs conda for experienced developers, and neither should those POVs drive the end-user recommendations, and IMO more modern tools should be taken into account (I'm thinking of pixi) even if core developers stay on using their own preferences due to muscle memory).

So, I'm with @eteq here on ditching conda in favour of minforge (even though my own bikeshed is pip-coloured).

**Comment by pllim** [comment]
I took the liberty to push a follow-up commit here in response to review from @bsipocz . Hope you don't mind, @eteq !

Does it look less confusing now?

File: index.html [hunk 2]
```diff
@@ -117,13 +117,10 @@
...
+        There are a number of ways of installing the latest released version of the astropy core package. If you normally use <code>pip</code> to install Python packages, you can do:
         <p/><pre>pip install astropy[recommended] --upgrade</pre>
-        If instead you normally use <code>conda</code>, you can do:
-        <p/><pre>conda install -c conda-forge astropy</pre>
-        or using <code>update</code> instead of <code>install</code> if <code>astropy</code> is already installed.
         The astropy core package is also available in a number of other package managers, so be sure to check your preferred one!
-	<p>More detailed <a href="http://astropy.readthedocs.io/en/stable/install.html">installation instructions</a> (e.g., for building from source code locally) are in the documentation.</p>
+	<p>More detailed <a href="http://astropy.readthedocs.io/en/stable/install.html">installation instructions</a> (e.g., for installing in a different way or building from source) are in the package documentation.</p>
```
Comment (taldcroft) [inline index.html:123]:
```suggestion
	<p>Please see the <a href="https://docs.astropy.org/en/latest/install.html"><code>astropy</code> installation guide</a> for important details. This guide covers creating a Python environment, installing with <code>pip</code> or <code>conda</code>, building from source, requirements, and testing.</p>
```

File: index.html [hunk 3]
```diff
@@ -117,13 +117,10 @@
 
 	<section class="install">
 		<h1 id="install-astropy">Install Astropy<a class="paralink" href="#install-astropy" title="Permalink to this headline">¶</a></h1>
-        There are a number of ways of installing the latest version of the astropy core package. If you normally use <code>pip</code> to install Python packages, you can do:
+        There are a number of ways of installing the latest released version of the astropy core package. If you normally use <code>pip</code> to install Python packages, you can do:
         <p/><pre>pip install astropy[recommended] --upgrade</pre>
-        If instead you normally use <code>conda</code>, you can do:
-        <p/><pre>conda install -c conda-forge astropy</pre>
-        or using <code>update</code> instead of <code>install</code> if <code>astropy</code> is already installed.
         The astropy core package is also available in a number of other package managers, so be sure to check your preferred one!
```
Comment (taldcroft) [inline index.html:122]: What is the status of "other package managers"? Is this now just Debian-based linux package managers? AFAIK we no longer support any Mac package managers.

File: index.html [hunk 4]
```diff
@@ -117,13 +117,10 @@
 
 	<section class="install">
 		<h1 id="install-astropy">Install Astropy<a class="paralink" href="#install-astropy" title="Permalink to this headline">¶</a></h1>
-        There are a number of ways of installing the latest version of the astropy core package. If you normally use <code>pip</code> to install Python packages, you can do:
+        There are a number of ways of installing the latest released version of the astropy core package. If you normally use <code>pip</code> to install Python packages, you can do:
```
Comment (taldcroft) [inline index.html:120]:
```suggestion
        There are a number of ways of installing the latest release of the astropy core package. If you normally use <code>pip</code> to install Python packages, you can do:
```

File: index.html (hunk 3 above)
Comment (pllim) [reply index.html:122]: Once in a while, we get issues from random distros (e.g., #14166) so they are out there even if we might not be aware.

**Comment by pllim** [comment]
Thanks for the review, @taldcroft ! I accepted all your suggestions.

**Comment by pllim** [comment]
@taldcroft , what do you think? Good to merge?

**Review by taldcroft — APPROVED** [review]
Looks good!

**Comment by pllim** [comment]
@bsipocz , conda install instruction is gone. Can you please re-review as well. Thanks!

**Review by bsipocz — APPROVED** [review]
Thanks, looks good now.

File: index.html (hunk 3 above)
Comment (eteq) [reply index.html:122]: @taldcroft , just fyi: Red Hat and Arch also package astropy (https://rpmfind.net/linux/rpm2html/search.php?query=python-astropy and https://archlinux.org/packages/extra/x86_64/python-astropy/), but they're basically just silently working (I can speak for the Arch one at least, which has worked perfectly for ~4 years.)

**Pull request merged**